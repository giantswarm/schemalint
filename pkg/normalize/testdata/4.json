{
"$schema": "http://json-schema.org/schema#",
"type": "object",
"properties": {},
"$id": "http://example.com/myschema"
}