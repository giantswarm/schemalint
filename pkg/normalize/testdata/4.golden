{
    "$id": "http://example.com/myschema",
    "$schema": "http://json-schema.org/schema#",
    "type": "object",
    "properties": {}
}
